profile/README.md

```diff
@@ -18,9 +18,9 @@
 
 </div>
 
-
-Raise an [issue to join the EddieHub GitHub organisation](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization).
-
-[![Join our Discord community here](https://camo.githubusercontent.com/c26e422415c93d08e0366cc8f0e8fd43927668a0acac5875038e923a11dd971f/68747470733a2f2f7777772e66726565706e676c6f676f732e636f6d2f75706c6f6164732f646973636f72642d6c6f676f2d706e672f636f6e636f7572732d646973636f72642d6361727465732d766f6575782d666f72746e6974652d6672616e63652d362e706e67)](http://discord.eddiehub.org)   
-
-[![Subscribe our YouTube channel here](https://camo.githubusercontent.com/508aaf5f8583bd1a8e080f224b9cde5a017554eae8b5c318d936a6188f3c3c74/68747470733a2f2f7777772e6672656569636f6e73706e672e636f6d2f7468756d62732f796f75747562652d6c6f676f2d706e672f68642d796f75747562652d6c6f676f2d706e672d7472616e73706172656e742d6261636b67726f756e642d32302e706e67)](https://www.youtube.com/user/eddiejaoude)
+<p align="center">Raise an <a href="https://github.com/EddieHubCommunity/support/issues/new?assignees=&amp;labels=invite+me+to+the+organisation&amp;template=invitation.yml&amp;title=Please+invite+me+to+the+GitHub+Community+Organization">issue to join the EddieHub GitHub organisation</a>.</p>
+    
+<p align="center">
+    <a align="center" href="http://discord.eddiehub.org"><img src="https://camo.githubusercontent.com/c26e422415c93d08e0366cc8f0e8fd43927668a0acac5875038e923a11dd971f/68747470733a2f2f7777772e66726565706e676c6f676f732e636f6d2f75706c6f6164732f646973636f72642d6c6f676f2d706e672f636f6e636f7572732d646973636f72642d6361727465732d766f6575782d666f72746e6974652d6672616e63652d362e706e67" width="50" alt="Join our Discord community here" /></a>
+    <a align="center" href="https://www.youtube.com/user/eddiejaoude"><img src="https://camo.githubusercontent.com/508aaf5f8583bd1a8e080f224b9cde5a017554eae8b5c318d936a6188f3c3c74/68747470733a2f2f7777772e6672656569636f6e73706e672e636f6d2f7468756d62732f796f75747562652d6c6f676f2d706e672f68642d796f75747562652d6c6f676f2d706e672d7472616e73706172656e742d6261636b67726f756e642d32302e706e67" width="50" alt="Subscribe our YouTube channel here" /></a>
```

Comment:
Perhaps @krishguptadev save the icons in an image folder, so the url isn't so long, and is saved on this repo.

Reply:
I think it is good there.

Reply:
We don't need to add to them to the repo 🤔

Reply:
Ok, I just thought a url like this: `https://camo.githubusercontent.com/508aaf5f8583bd1a8e080f224b9cde5a017554eae8b5c318d936a6188f3c3c74/68747470733a2f2f7777772e6672656569636f6e73706e672e636f6d2f7468756d62732f796f75747562652d6c6f676f2d706e672f68642d796f75747562652d6c6f676f2d706e672d7472616e73706172656e742d6261636b67726f756e642d32302e706e67` was a bit long. Let's see what others say.

Reply:
We are waiting for other people's opinions on this right? @CodingSpecies